src/webview/android/kotlin/proguard-wry.pro

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020-2023 Tauri Programme within The Commons Conservancy
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-License-Identifier: MIT
+
+-keep class {{package}}.* {
+  native <methods>;
+}
+
+-keep class {{package}}.WryActivity {
```

Comment:
```suggestion
-keep class {{package}}. TauriActivity {
```

After this change everything works fine

Reply:
This PR is pending for https://github.com/tauri-apps/wry/pull/926 to be merged where `TauriActivity` will be changed to `WryActivity`

Reply:
You can update it now.